default/stringtables/en.txt

```diff
@@ -2255,6 +2255,29 @@ A:
 ## Intro screen
 ##
 
+MAIN_MENU_TITLE
+[[INTRO_WINDOW_TITLE]]
+
+MAIN_MENU_SHORT_DESC
+[[INTRO_WINDOW_TITLE]]
+
+MAIN_MENU_TEXT
+'''This screen shows the [[INTRO_WINDOW_TITLE]] for starting a FreeOrion session. The [[INTRO_BTN_PEDIA]] button opens the [[encyclopedia PEDIA_TITLE]] page; use this to get in-game help by using the Pedia button or from the right-click menu on various elements.
+
+The following buttons are available:
+
+[[encyclopedia CONTINUE_TITLE]] - Continues with the last saved game.
+[[encyclopedia SINGLE_PLAYER_TITLE]] - Start a new game for 1 player to play against AI opponents. Various game options can be set before creating a Galaxy and starting the game.
+[[encyclopedia QUICK_START_TITLE]] - Start a new single player game using the options set by the last single player game created.
+[[encyclopedia MULTI_PLAYER_TITLE]] - Start playing a multiple player game by joining an existing multi player game or by creating a new game for other players.
+[[encyclopedia LOAD_GAME_TITLE]] - Load a previously saved game.
```

Comment:
clarify single player

Reply:
Is this comment for me? I don't understand what needs clarifying: "a game for 1 player to play against AI opponents". This is a link to another article with more details.

Reply:
I mean you can't use that button to load a multiplayer game; just single player saves.

Reply:
Ok - I have changed the texts for loading saved games (en.txt)